dace/transformation/interstate/state_elimination.py

```diff
@@ -646,3 +646,47 @@ def apply(self, _, sdfg: SDFG):
         b: SDFGState = self.state_b
         edge = sdfg.edges_between(a, b)[0]
         edge.data.condition = CodeBlock("1")
+
+
+class FalseConditionElimination(transformation.MultiStateTransformation):
+    """
+    If a state transition condition is always true, removes condition from edge.
```

Comment:
Is the docstring correct? The transformation name sounds like it handles conditions that are never satisfied.

Reply:
Fixed